assets/js/modules/ads/components/settings/SettingsView.js

```diff
@@ -57,9 +59,41 @@ export default function SettingsView() {
 
 	const isPaxView = paxEnabled && ( paxConversionID || extCustomerID );
 
+	const isConversionTrackingEnabled = useSelect( ( select ) => {
+		if ( ! iceEnabled ) {
+			return false;
+		}
+
+		return select( CORE_SITE ).isConversionTrackingEnabled();
+	} );
+
 	return (
 		<Fragment>
 			<AdBlockerWarning />
+			{ iceEnabled && (
+				<div className="googlesitekit-settings-module__meta-item">
+					{
+						// This is required to add correct spacing if the <AdBlockerWarning> component is present.
+						isAdBlockerActive && <br />
+					}
```

Comment:
This is not how it should be done. Let's update the `AdBlockerWarning` component to have an optional `className` property that we can use to pass `googlesitekit-settings-module__meta-item`.

Reply:
@eugene-manuilov I've addressed this in my latest few commits, thank you. Re-assigning to you.